visualizer/IdealGraphVisualizer/SourceRepository/src/org/graalvm/visualizer/source/impl/ui/StackViewTopComponent.java

```diff
@@ -702,10 +702,17 @@ private void updateComboContents(NodeStack locs) {
                 }
             }
         }
-        NodeStack.Frame head = locs.bottom();
-        ChainNode node = new ChainNode(head, head.getLookup().lookup(Node.class),
-                                       head.getNested() == null ? Children.LEAF : new ChainChildren(head));
+        final int indentLevel = 24;
```

Comment:
This constant can change, but no more than to [50 as limited here](https://github.com/oracle/graal/blob/f2b350f312f95ff366c149c05950ef19de72e14e/visualizer/IdealGraphVisualizer/SourceRepository/src/org/graalvm/visualizer/source/impl/ui/StackViewTopComponent.java#L149).